src/io/flutter/preview/RenderHelper.java

```diff
@@ -222,27 +216,21 @@ private void scheduleRendering() {
     this.height = height;
     this.listener = listener;
   }
+
+  Module getModule() {
+    return pubRoot.getModule(project);
```

Comment:
Might pubRoot be null?

Reply:
No, but marking fields and parameters explicitly with `@NotNull` is a good idea.